lastuser_ui/views/sms.py

```diff
@@ -49,6 +49,32 @@ def send_message(msg):
                     flash(_("Message could not be sent"), 'danger')
             except requests.ConnectionError:
                 flash(_("The SMS delivery engine is not reachable at the moment. Please try again"), 'danger')
+    else:
+        if len(msg.phone_number) != 13:
```

Comment:
Non-Indian numbers could be any length. Only Indian mobile numbers are 13 digit (`'+91'` + 10 digits).